WordPress/Classes/Plugins/Views/PluginDetailsView.swift

```diff
@@ -124,6 +176,28 @@ struct PluginDetailsView: View {
         .listRowInsets(EdgeInsets(top: 0, leading: 16, bottom: 0, trailing: 16))
     }
 
+    @ViewBuilder
+    private func uninstalling() -> some View {
```

Comment:
(nit) change to property, rename to `uninstallingView` or something like is as "uninstalling" isn't a noun.